manifest: bump Mbed TLS to v4.1.1 and TF-PSA-Crypto to v1.1.1 (#113047)
zephyrproject-rtos/zephyr: doc/releases/migration-guide-4.5.rst

  as an alias to the latter for backward compatibility, but it will be removed in future
  releases.

> * Mbed TLS has been upgraded to version 4.1.1. Release notes can be found

Reviewer: ```suggestion
* Mbed TLS was updated to version 4.1.1. Release notes can be found
```

Reviewer: \+ also especially mention the updates in the release notes?

  as an alias to the latter for backward compatibility, but it will be removed in future
  releases.

* Mbed TLS was updated to version 4.1.1. Release notes can be found
  `here <https://github.com/Mbed-TLS/mbedtls/releases/tag/mbedtls-4.1.1>`_.

> * TF-PSA-Crypto was udpated to version 1.1.1. Release notes can be found

Reviewer 2: udpated -> updated
